Gemfile.lock

```diff
@@ -1,8 +1,8 @@
 PATH
   remote: .
   specs:
-    fluent-plugin-kubernetes_metadata_filter (2.4.6)
-      fluentd (>= 0.14.0, < 1.10)
+    fluent-plugin-kubernetes_metadata_filter (2.5.0)
+      fluentd (>= 1.10)
```

Comment:
why not just adjust the range to include `1.11` ?

Reply:
restrict upper bound to less then 1.11